OpenRA.Mods.Common/Traits/Render/WithSpriteControlGroup.cs

```diff
@@ -0,0 +1,86 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.Graphics;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits.Render
+{
+	[Desc("Renders Ctrl groups using pixel art.")]
+	public class WithSpriteControlGroupInfo : ITraitInfo
```

Comment:
This and the text trait are both decorations, so should follow the naming, property, and style conventions of `WithDecoration` and `WithTextDecoration`.